helm/cluster/templates/bastion/_kubeadmconfigtemplatespec.yaml

```diff
@@ -0,0 +1,11 @@
+{{- define "cluster.internal.bastion.kubeadmconfigtemplate.spec" }}
```

Comment:
Template that renders kubeadm template spec for bastion machines. This is a named template because because we have to be able to calculate its hash (see above hash explanation) in order to detect changes in the spec and do proper upgrades.